internal/internal_worker.go

```diff
@@ -410,7 +425,6 @@ func newSessionWorker(service workflowservice.WorkflowServiceClient, params work
 	params.TaskQueue = creationTaskqueue
 	// Although we have session token bucket to limit session size across creation
 	// and recreation, we also limit it here for creation only
-	params.ConcurrentActivityExecutionSize = maxConcurrentSessionExecutionSize
```

Comment:
Why is this line being removed?

Reply:
Because the field doesn't exist any more, the session bucket ought to be responsible for any limiting specific to sessions

Reply:
Hm  need to think about this a bit, but regardless isn't the comment above wrong now?

Reply:
Removed

Reply:
Looking at this change, it looks like you always giving the session `creationWorker` the activity workers slot supplier? That doesn't seem right. Shouldn't it have a fixed slot supplier that is configured by `maxConcurrentSessionExecutionSize `?

Reply:
Yes, that makes sense. This seems dumb on my part but I didn't even realize two workers were being made here.

Reply:
Sounds good, can you please add the comment back then